packages/xstate-dev/panel/src/utils/getItemString.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { Iterable } from 'immutable';
```

Comment:
I don't think we should support `immutable`.

Reply:
Agree!
I am not sure what it's being used for (extracted it from elsewhere). 
Will try to remove it and update the PR